resources/views/layouts/app.blade.php

```diff
@@ -1,23 +1,25 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ja">
     <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Laravel Quickstart - Basic</title>
-        <!-- Fonts -->
+        {{-- <!-- Fonts --> --}}
         <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.4.0/css/font-awesome.min.css" rel='stylesheet' type='text/css'>
-        <!-- Styles -->
-        <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">
+        {{-- <!-- Styles --> --}}
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
     </head>
     <body>
-        <nav class="navbar navbar-default">
+        <nav class="navbar navbar-expand-lg navbar-default bg-light">
             <div class="container">
-                <!-- navbar  -->
+                {{-- <!-- navbar  --> --}}
                 <div class="navbar-header">
-                    <a class="navbar-brand" href="{{ url('/')  }}">Task List</a>
+                    <a class="navbar-brand text-secondary" href="{{ route('tasks.index')  }}">Task List</a>
```

Comment:
```suggestion
                    <a class="navbar-brand text-secondary" href="{{ route('tasks.index') }}">Task List</a>
```
エコー文（{{ }})は処理の前後に半角スペース１でお願いします。

Reply:
承知しました。
余分な記述を削除しました。